mlir/utils/jenkins/Jenkinsfile

```diff
@@ -857,6 +866,18 @@ pipeline {
                             }
                         }
                     }
+                    //stage("Test Fusion") {
```

Comment:
Why's this commented out?

Reply:
Because we don't have fusion tuning DB yet, it will fail in nightly CI. I'll enable it after the weekly generates fusion tuning DBs.

Reply:
Ah